tests/mgbench/compare_results.py

```diff
@@ -85,39 +85,35 @@ def compare_results(results_from, results_to, fields):
                 if group == "__import__":
                     continue
                 for scenario, summary_to in scenarios.items():
-                    summary_from = recursive_get(
-                        results_from, dataset, variant, group, scenario,
-                        value={})
-                    if len(summary_from) > 0 and \
-                            summary_to["count"] != summary_from["count"] or \
-                            summary_to["num_workers"] != \
-                            summary_from["num_workers"]:
+                    if scenario == "expansion_4" or scenario == "expansion_4_with_filter":
+                        continue
```

Comment:
What's the problem with scenario_4, it might be that it's too slow. If so, that's fine but that should be solved by additional config or somehow similar, e.g. a black list of scenarios